### ISSUE TITLE
Admin UI: Update color contrast for text

### ISSUE THREAD

**Issue by ryelle** [Automattic/jetpack#6532]
There are a few places on the dashboard where the color contrast of text is too low to be comfortably readable according to accessibility guidelines. For example, [WordPress core aims for 4.5+ color contrast.](https://make.wordpress.org/core/handbook/best-practices/coding-standards/accessibility-coding-standards/#color-contrast)

These screenshots indicate where the contrast falls below 4.5:

![dashboard](https://cloudup.com/iD3Rzl1kQfY+)

![settings](https://cloudup.com/iX0p2AC2w26+)

(the rest of settings text was fine)

I used this chrome extension to flag for contrast issues: http://wave.webaim.org/

You can use this site to play with contrast, if you need to: http://contrast-finder.tanaguru.com/

There are a handful of calypso PRs doing this right now, see https://github.com/Automattic/wp-calypso/pull/11464, https://github.com/Automattic/wp-calypso/pull/11550, https://github.com/Automattic/wp-calypso/pull/11518

**Comment by eliorivero** [comment]
@MichaelArestad we should take a look at this as Calypso makes progress.

I checked the colors of the light blue with white text button (our save button) 
https://snook.ca/technical/colour_contrast/colour.html#fg=FFFFFF,bg=00AADC
and it doesn't comply with any contrast standard.

<img width="857" alt="captura de pantalla 2017-03-15 a las 17 36 56" src="https://cloud.githubusercontent.com/assets/1041600/23969695/18d7324a-09a6-11e7-845d-2307213664bc.png">
 
Maybe we should consider introducing a new stylesheet to be optionally activated for more contrast? So that we don't change our branding and be kind to all our users at the same time.

**Comment by MichaelArestad** [comment]
@ryelle Of these, I see a few we can tackle this release, but others we should probably stick with our branding (buttons). 

Easily fixable:
- [ ] "Paid" label on dashItems could have a white background even when the card is disabled
- [ ] footer links could be a shade or two darker
- [ ] maaaybe some of the stray link colors (nav) because we could just remove the styling there defaulting to core link colors

Problematic due to branding and requiring a bit more discussing/work to fix:
- [ ] I am unsure about the banners, but we switched to a different darker variant. I'm pretty sure the other ones don't meet AA standards
- [ ] button colors. I've looked at this a bunch of times with Hugo and this is a toughy. If we make them dark enough to actually meet standards (Core currently has a bit of a hack that I don't think technically meets standards via text shadow) it gets pretty dang dark or off-brand. I'm at a loss of what to do here unless we finally implement CSS color variables and work up some fancy color schemes including a nice accessible scheme.

**Comment by MichaelArestad** [comment]
It looks like the red will be simpler to fix (I think) because it didn't take much to pass. The blue had to get quite a bit darker to barely pass. The modified (darker) buttons are the left ones:

![image](https://cloud.githubusercontent.com/assets/1123119/23975244/2b49f724-09a5-11e7-988e-3f4fe5b40125.png)

**Comment by MichaelArestad** [comment]
Yeah. This banner I don't know what to do with as is. The top one is how dark it has to get with the white text to pass:

![image](https://cloud.githubusercontent.com/assets/1123119/23975460/7fb8bb6e-09a6-11e7-9193-7ab4979b8a2d.png)

**Comment by ryelle** [comment]
I didn't know about the `settings-overhaul` branch when I made most of these issues, so since then I've been trying to work off that one instead. Here are the updated contrast tests — the banner text passes now, but now the button text doesn't. Otherwise not much has changed. ~~One thing I didn't notice before is the yellow notice text, that definitely doesn't pass - so I wonder about the other notices, too.~~ You're on top of this, I see :)

![screen shot 2017-03-15 at 7 25 42 pm](https://cloud.githubusercontent.com/assets/541093/23975195/bd548890-09b5-11e7-92ad-b18a733b28b4.png)

![screen shot 2017-03-15 at 7 25 42 pm](https://cloud.githubusercontent.com/assets/541093/23975196/be7e8c02-09b5-11e7-9ec8-826d0599cbd8.png)

As we're following Calypso colors, could we just go with whatever blue is decided on from the discussion in https://github.com/Automattic/wp-calypso/pull/11550?

**Comment by MichaelArestad** [comment]
For the white banners and secondary buttons, we can def bump the text contrast. Will see if I can some of these in during the next couple days. I think they're likely pretty simple changes.

**Comment by scottsweb** [comment]
Closing this issue as the UI has move on since this point. I will note that a couple of the footer links and chart labels still fail, but the rest have been improved over time.
